src/ui/public/routes/route_manager.d.ts

```diff
@@ -30,9 +30,11 @@ interface RouteConfiguration {
   resolve?: object;
   template?: string;
   k7Breadcrumbs?: (...args: any[]) => Breadcrumb[];
+  requireUICapabilities?: string;
```

Comment:
I'd expect `requireUICapabilities` to accept `string[]` rather than `string`. How do you feel about renaming this to `requireUICapability`?

Reply:
Yeah, the naming is rather awkward, how would you feel about leaving it named `requireUICapabilities` and making it accept a `string | string[]`?

Reply:
I like it!

Reply:
If more than one is specified, should they be treated like `.some` or `.all`?

Reply:
I was initially thinking `.all`, but I could see the potential for `.some`... let's just leave it singular and change the name since we don't have a plural use-case at the moment.